### PR TITLE
frontend: prefer named exports

### PR DESCRIPTION
**Pull request by thisconnect**
Refactored TypeScript files to use named exports instead of default exports to improve code readability and facilitate better auto-completion in IDEs. This change also enhances the consistency and maintainability of the codebase by explicitly defining exported members.

### CODE REVIEW

**Review by shonsirsha — APPROVED** [review]
untested LGTM. Thanks

**Comment by thisconnect** [comment]
rebased

File: frontends/web/src/components/toast/Toast.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    } = this.props;
+    const { active } = this.state;
+    return (
+      <div
+        className={[style.toast, style[theme], active ? style.active : '', withGuide ? style.shifted : ''].join(' ')}>
+        <p>{children}</p>
+      </div>
+    );
+  }
+}
```
Comment (thisconnect) [inline frontends/web/src/components/toast/Toast.jsx:54]: accidentally pushed this while rebasing, ci didn't mind  no error. Seems tsconfig node moduleResolution just finds the toast module and doesn't care there there is potential conflict. Removing...

Comment (thisconnect) [reply frontends/web/src/components/toast/Toast.jsx]: PR udpated

**Pull request merged**